Themes/default/languages/Admin.english.php

```diff
@@ -153,7 +153,7 @@
 $txt['ban_description'] = 'Here you can ban troublesome people either by IP, hostname, username, or email.';
 $txt['ban_add_new'] = 'Add new ban';
 $txt['ban_banned_entity'] = 'Banned entity';
-$txt['ban_on_ip'] = 'Ban on IP (e.g. 192.168.10-20.*)';
+$txt['ban_on_ip'] = 'Ban on IP (e.g. 19.2.168.10-20.*)';
```

Comment:
My interpretation of 19.2.168.10-20.* that the result is
19.2.168.10 is the lower bound of ip address and
20.255.255.255 the upper bound of the ip address range.

Another interpretation of this could be lower bound 19.2.168.10 and upper 19.2.168.20.

When the last one is the expected behavior i would recommand to change this.

The reading of this is to far complex
worst case (in ipv6): 0011-ffff:0031-ffff:0000-0022:0000-0011:0011-*:0031-*:0011-*:0000-*
my version: 0011:0031::0011:0031:0011:0000-ffff:ffff:0022:0011:*

advantage on both side of the - we got valid ip or not --> less complex logic
I should mention that this logic is implemented in my pr of https://github.com/SimpleMachines/SMF2.1/pull/3222